src/main/scala-2/com/evolutiongaming/random/Random.scala

```diff
@@ -100,29 +137,34 @@ object Random {
       (State(seed1, random), a)
     }
 
-    def int = apply(random.int)
+    def int: (State, Int) = apply(random.int)
 
-    def long = apply(random.long)
+    def long: (State, Long) = apply(random.long)
 
-    def float = apply(random.float)
+    def float: (State, Float) = apply(random.float)
 
-    def double = apply(random.double)
+    def double: (State, Double) = apply(random.double)
   }
 
-  object State { self =>
+  object State {
 
     type Type[A] = (State, A)
 
+    /** Create an instance of [[Random.State]] based on [[Clock]].
```

Comment:
full path to `Clock` fixes the compilation issue
```suggestion
    /** Create an instance of [[Random.State]] based on [[cats.effect.Clock]].
```